### DISCUSSION TITLE
I'm new to this. I need help adding an API.

### DISCUSSION THREAD

**Discussion by CenTexSlim** [danielmiessler/Fabric#530] [Q&A] [answered]
I am new to working with the CLI. I'm using PowerShell. I have Fabric installed and updated. I'm using what I can see in code and discussions, as well as ChatGPT to ask for general directions when interacting with the CLI.

At setup I added my open AI API key.  I now, after everything is set up and running,  have an API key for yt.  What commands do I use to add that to fabric?

**Comment by snafu4** [comment, marked as answer]
fabric --setup
